### PR TITLE
[onert] Introduce Loss operation

### PR DESCRIPTION
**Pull request by ragmani**
This commit introduce Loss operation.
  - Introduce Loss
  - Change Operation and ITrainableOperation to inherit virtual IOperation

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
@nnfw-bot test onert-x64-release

File: runtime/onert/core/include/ir/operation/Loss.h [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    CATEGORICAL_CROSSENTROPY
+  };
+
+  // NOTE It is not yet determined how to get the information of the previous activation.
+  //      If it is determined to get from the user, we have to consider whether to merge
+  //      this enum class and the `Type` enum class.
+  enum class PreActivationType
+  {
+    SOFTMAX
+  };
```
Comment (glistening) [inline runtime/onert/core/include/ir/operation/Loss.h:50]: @ragmani I don't understand why this is required. Could you please remove this and put the skeleton only in this PR? Let's review this in another PR.

File: runtime/onert/core/include/ir/operation/Loss.h [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (glistening) [inline runtime/onert/core/include/ir/operation/Loss.h:1]: I told @ragmani, why `ir/operation/Loss.h` needs to be introduced. I agreed to introduce this file:

- there are some required methods from `ir/operation`
- to be consistent

**Review by glistening — DISMISSED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by glistening** [comment]
@ragmani Please rebase.

**Comment by ragmani** [comment]
@nnfw-bot test onert-cross-release

**Comment by ragmani** [comment]
@nnfw-bot test onert-x64-release

**Review by glistening — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**